packages/adminrouter/extra/src/nginx.master.conf

```diff
@@ -368,6 +349,7 @@ http {
                 auth.access_mesosdns_endpoint();
             }
             include common/proxy-headers.conf;
+            proxy_set_header Authorization "";
```

Comment:
I like naming of grouped directives as `conf` files. Maybe we could name also this one (I understand it is only one directive), for example, `clear-authorization.conf`?